content/operate/kubernetes/deployment/helm.md

```diff
@@ -0,0 +1,112 @@
+---
+Title: Install Redis Enterprise Helm chart
+alwaysopen: false
+categories:
+- docs
+- operate
+- kubernetes
+description: Install the Redis Enterprise for Kubernetes version 7.8.Wisconsin using helm charts.
+linkTitle: Helm
+weight: 11
+---
+
+Helm charts provide a simple way to install the Redis Enterprise for Kubernetes operator in just a few steps. For more information about Helm, go to [https://helm.sh/docs/](https://helm.sh/docs/).
+
+{{<note>}} This feature is currently in public preview and is not supported on production workloads. Only new installations of the Redis operator are supported at this time. The steps for [creating the RedisEnterpriseCluster (REC)]({{<relref "operate/kubernetes/deployment/quick-start/#create-a-redis-enterprise-cluster-rec">}}) and other custom resources remain the same.{{</note>}}
+
+## Prerequisites
+
+- A [supported distribution]({{< relref "/operate/kubernetes/reference/supported_k8s_distributions.md" >}}) of Kubernetes.
+- At least three worker nodes.
+- [Kubernetes client (kubectl)](https://kubernetes.io/docs/tasks/tools/).
+- [Helm 3.10 or later](https://helm.sh/docs/intro/install/).
+
+## Install
+
+1. Add the `redis` repository.
+
+    ```sh
+    helm repo add redis-enterprise https://helm.redis.io/
+    ```
+
+1. Install the Helm chart into a new namespace.
+
+    ```sh
+    helm install <operator-name> redis/redis-enterprise-operator \
+        -- version <release-name> \
+        -- namespace <namespace-name> \
+        -- create-namespace
+    ```
+
+To install with Openshift, add `--set openshift.mode=true`.
+
+To monitor the installation add the `--debug` flag. The installation runs several jobs synchronously and may take few minutes to complete.
+
+### Install from local directory
+
+1. Find the latest release on the [redis-enterprise-k8s-docs Github](https://github.com/RedisLabs/redis-enterprise-k8s-docs/releases) and download the `tar.gz` source code into a local directory.
+
+1. Install the Helm chart from your local directory.
+
+    ```sh
+    helm install <release-name> <path-to-chart> \
+        -- namespace <namespace-name> \
+        -- create-namespace
+    ```
+
+To install with Openshift, add `--set openshift.mode=true`.
+
+To monitor the installation add the `--debug` flag. The installation runs several jobs synchronously and may take few minutes to complete.
+
+### Specify values during install
+
+1. View configurable values with `helm show values redis/redis --version <release-name>`.
+
+1. Install the Helm chart, overriding specific value defaults using `--set`.
+
+    ```sh
+    helm install <operator-name> redis/redis-enterprise-operator \
+        -- version <release-name> \
+        -- namespace <namespace-name> \
+        -- create-namespace
+        --set <key1>=<value1> \
+        --set <key2>=<value2>
+    ```
+
+### Install with values file
```

Comment:
This section is largely repeating the previous one, maybe we should just mention that the values may be overridden either by the `--set` flag or via `--values` file?